admin-dev/themes/new-theme/js/stock-page/store/modules/products.js

```diff
@@ -0,0 +1,117 @@
+import * as types from '../mutation-types';
+import Vue from 'vue';
+import VueResource from 'vue-resource';
+import { showGrowl } from '../utils/growl';
+
+Vue.use(VueResource);
+
+// initial state
+const state = {
```

Comment:
the right behavior of state is to never been able to change or be updated? You probably want to use `let` or `var`